src/chaboche.jl

```diff
@@ -99,49 +167,89 @@ function integrate_material!(material::Chaboche)
                                           cumeq = cumeq,
                                           jacobian = jacobian)
     material.variables_new = variables_new
+    return nothing
 end
 
-function create_nonlinear_system_of_equations(material::Chaboche)
+"""
+    create_nonlinear_system_of_equations(material::Chaboche)
+
+Create and return an instance of the equation system for the delta form of the
```

Comment:
perhaps `incremental form` instead of `delta form`?

Reply:
Sure, maybe clearer. Changed.